extra-libs/zimg/autobuild/defines

```diff
@@ -2,10 +2,7 @@ PKGNAME=zimg
 PKGSEC=libs
 PKGDEP="gcc-runtime"
 PKGDES="Scaling, colorspace conversion, and dithering library"
+PKGBREAK="vapoursynth<=48"
```

Comment:
The newest version is obviously not 48. Check with `apt list vapoursynth`.

Reply:
Sorry for that, I took it for granted that put the VER here but not append the REL, I'll fix it soon. Thanks~

Reply:
Resolved in 4d084c8.